--- disk_usage.py	2023-03-26 21:32:01.896000000 +0800
+++ disk_usage_fixed.py	2023-03-26 20:20:11.458000000 +0800
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 import shutil
-
+import sys
 def check_disk_usage(disk,min_absolute,min_percent):
     """Returns True if there is enough free disk space, false otherwise."""
     du = shutil.disk_usage(disk)
@@ -13,9 +13,9 @@
     return True
 
 #check for at least 2 GB and 10% free
-if not check_disk_usage("/", 2*2*30,10):
+if not check_disk_usage("/", 2,10):
     print("ERROR: Not enough disk space")
-    return 1
+    sys.exit(1)
 
 print("Everything ok")
-return 0
+sys.exit(0)
